Numbers_Only_Filter_Example.c

```diff
@@ -0,0 +1,113 @@
+/*********** Numbers Only Filter *****************
+ * This is simple example shows how to implement a
+ * filter for Efl.Ui.Text that accepts only numbers
+ * and filter out other characters.
+ *
+*/
+
+#define EFL_EO_API_SUPPORT 1
+#define EFL_BETA_API_SUPPORT 1
+
+#include <Eina.h>
+#include <Elementary.h>
+#include <Efl_Ui.h>
+
+/******************** Callbacks *********************/
+
+#define IS_DIGIT(x) ((x >= '0') && (x <= '9'))
+
+static void
+_ui_text_changed_user_pre_cb(void *data, const Efl_Event *event)
+{
```

Comment:
The biggest problem is that 
Efl2_Ui_Text_Change_Info content is ** const char * ** so user can not modify it.

Reply:
Other problem if user wants to use a larger buffer(Efl2_Ui_Text_Change_Info.Content), how he is expected to manage memory for old buffer?